transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystemRootResourceDefinition.java

```diff
@@ -155,6 +155,13 @@
             .setXmlName(Attribute.DEFAULT_TIMEOUT.getLocalName())
             .setAllowExpression(true).build();
 
+    public static final SimpleAttributeDefinition MAXIMUM_TIMEOUT = new SimpleAttributeDefinitionBuilder(CommonAttributes.MAXIMUM_TIMEOUT, ModelType.INT, true)
+            .setValidator(new IntRangeValidator(300))
```

Comment:
Why 300?

Reply:
I assume that at least the maximum_timeout should be larger than the default value of timeout which is 300. Actually the maximum_timeout might be a little smaller than Integer.MAX_INT.